### PR TITLE
Support non-0 LUN for GetSensorReadingCmd

### LINKED ISSUE #56: RemoteLUN!=0 is not supported
I am testing out a new version of a BMC. This library is giving different readings from ipmitool. In the changelog for this version, they claim that ipmitool>=1.8.18 is required because "the issues with the older ipmitool was that the LUN of the sensor was not passed around with the sensor number".

Might this library have a similar issue as the older versions of ipmitool which would explain the discrepancy? I would be very willing to believe that the behavior exhibited by this BMC and by ipmitool is not according to spec.

### PR DESCRIPTION
**Pull request by lukeyeager**
Close #56

### CODE REVIEW

**Comment by lukeyeager** [comment]
@gebn Have you had a chance to take a look at this yet? FWIW, this patch has been working well for me.

File: pkg/ipmi/command.go [hunk 1]
```diff
@@ -33,6 +33,10 @@ type Command interface {
 	// otherwise need to have a no-op layer created.
 	Operation() *Operation
 
+	// Most commands use RemoteLUN 0 (LUNBMC). Others (such as GetSensorReading
```
Comment (gebn) [inline pkg/ipmi/command.go:36]: Can you rearrange this to begin with `RemoteLUN`, so we follow the [convention](https://go.dev/blog/godoc#:~:text=Notice%20this%20comment%20is%20a%20complete%20sentence%20that%20begins%20with%20the%20name%20of%20the%20element%20it%20describes.)?

File: pkg/ipmi/get_sensor_reading.go [hunk 2]
```diff
@@ -12,6 +12,9 @@ import (
 type GetSensorReadingReq struct {
 	layers.BaseLayer
 
+	// Use the value from FullSensorRecord.OwnerLUN.
```
Comment (gebn) [inline pkg/ipmi/get_sensor_reading.go:15]: The request was intended to be that layer in the UDP payload. What do you think about putting this field in the `GetSensorReadingCmd` struct?

**Review by gebn — COMMENTED** [review]
Thanks! Two minor comments. It's a shame we have to add essentially the same 3 lines to every command, however I can't think of a cleaner way to do this.

File: pkg/ipmi/get_sensor_reading.go (hunk 2 above)
Comment (lukeyeager) [reply pkg/ipmi/get_sensor_reading.go]: Sounds good to me. I had to rename the field so as to avoid overlapping with the function's name.

**Comment by lukeyeager** [comment]
> It's a shame we have to add essentially the same 3 lines to every command, however I can't think of a cleaner way to do this.

I agree. Golang has its upsides, but it also has downsides.

**Pull request merged**